### ISSUE TITLE
no complex valued fields?

### ISSUE THREAD

**Issue by AshtonSBradley** [CalculustJL/FourierSpaces.jl#14]
the fftw plans seem to use real planning only, even for complex initial conditions. Is the intention to disallow complex fields?

**Comment by vpuri3** [comment]
Yes, that was by design as my application at that time was concerned with solving real partial differential equations.

If you want, you can make another `Spaces.AbstractSpace` subtype for complex transform following https://github.com/CalculustJL/FourierSpaces.jl/blob/master/src/type.jl#L5-L30

**Comment by AshtonSBradley** [comment]
Thanks for the pointer. I have questions about the package - are there any benchmarks? You seem to flatten arrays, is there any performance loss? Do you have a plan in mind for future development or maintenance? Is there scope to mesh this with stochastic diffeq solving? My main issue with using it is that on the one hand the idea is great but on the other I have simpler code that works so I am trying to understand the advantages. Feel free to ignore but any advice appreciated

**Comment by vpuri3** [comment]
Thanks for your interest. There are no plans to extend CalculustJL at this point. My primary use for this package is data generation for my PhD research on ML and PDEs. I will add new functionality to this package in service of that goal from time to time, though I unfortunately cannot commit to lasting plan.

I frequently use this package in conjunction with OrdinaryDiffEq.jl, so StochasticDiffEq.jl should be ok too? You can give it a shot and open an issue if there are errors.
